Cabal/Distribution/Simple.hs

```diff
@@ -98,7 +98,8 @@ import Distribution.Text
 -- Base
 import System.Environment (getArgs, getProgName)
 import System.Directory   (removeFile, doesFileExist, getCurrentDirectory
-                          ,doesDirectoryExist, removeDirectoryRecursive)
+                          ,doesDirectoryExist, removeDirectoryRecursive
+                          ,makeAbsolute)
```

Comment:
cannot use. it's only in `1.2.2.0 (Mar 2015)` https://hackage.haskell.org/package/directory-1.3.2.0/changelog which is GHC-7.10.3...

Reply:
Ack